### PR TITLE
Merge trainers branch to examples

### PR DESCRIPTION
**Pull request by MrLYG**

### CODE REVIEW

**Comment by fostiropoulos** [comment]
The test are failing.

**Review by fostiropoulos — CHANGES_REQUESTED** [review]
The test are failing. Let's fix that before merging.

File: setup.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
         "torchvision==0.14.1",
         "tqdm==4.64.1",
         "tensorboardX==2.6",
+        "tensorboard==2.12.2",
```
Comment (fostiropoulos) [inline setup.py]: As we discussed, it should be removed.

**Review by fostiropoulos — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (MrLYG) [reply setup.py]: Yeah, we are working on that. and handle all TODOs

**Review by fostiropoulos — APPROVED** [review]

**Pull request merged**